packages/prci/src/utils/privacy-request-api.ts

```diff
@@ -1,5 +1,5 @@
+/* eslint-disable camelcase */
```

Comment:
@jboileau99 please try to avoid this kind of comment.

Favour `eslint-disable-next-line`, then blocks with `eslint-disable` AND `eslint-enable`, then, for the whole file like here.

see https://eslint.org/docs/latest/user-guide/configuring/rules#using-configuration-comments-1

You should also add a description explaining why you disabled the rule (with `--`).

It would be a good idea to add https://mysticatea.github.io/eslint-plugin-eslint-comments/ to avoid discussing this.